src/bonsai/websocket/lib/indexerWebsocket.ts

```
@@ -342,124 +435,69 @@ export class IndexerWebsocket {
 
   private _handleMessage = (messagePre: any) => {
     try {
-      const message = isWsMessage(messagePre);
+      // messagePre is already parsed by ReconnectingWebSocket, so we don't need to parse it again
+      const message = messagePre;
+
+      logBonsaiInfo('IndexerWebsocket', 'Raw message received', { messagePre, wsId: this.indexerWsId, });
```

ESLint

[error] Replace `·messagePre,·wsId:·this.indexerWsId,` with `⏎········messagePre,⏎········wsId:·this.indexerWsId,⏎·····`

(prettier/prettier)